automated install/basic-install.sh

```diff
@@ -669,7 +669,7 @@ installScripts() {
 }
 
 installConfigs() {
-  # Install the configs from /etc/.pihole to their various locations
+  # Install the configs from ${PI_HOLE_LOCAL_REPO} to their various locations
```

Comment:
Though, that said, if the install location was ever to change, it's probably a good idea to have the variable name in it's place :)